ArticleTemplates/assets/js/bootstraps/campaign.js

```diff
@@ -11,11 +11,11 @@ define([], function () {
     function initCampaign(campaign) {
       campaign.onSubmit(function () {
         const data = new FormData(campaign);
-        // TODO form submission URL
-        const request = new XMLHttpRequest('POST', '#');
-        request.onload(displayConfirmation.bind(null, campaign));
-        request.onerror(displayError.bind(null, campaign));
-        request.send(data);
+        const json = {};
+        for([key, value] of data.entries) {
+          json[key] = value;
+        }
+        location.hash = JSON.stringify(json);
```

Comment:
@aodhol like this?

Reply:
Could it be the location's query rather than hash?

Reply:
Whatever works 🤗  as long as it works on both platforms and the app layer catches the call without reloading the page, I'm all good.